Makefile

```
@@ -154,6 +163,41 @@ copy-coverage: ## Copy the Jest coverage directory from the docker container.
 test-mutation: ## Run mutation tests inside the docker container.
 	$(BUN_X) stryker run
 
+test-mutation-shard: ## Run mutation shard MUTATION_SHARD_INDEX of MUTATION_SHARD_TOTAL inside the docker container.
```

checkmake (0.3.2)

[warning] Target body for "test-mutation-shard" exceeds allowed length of 5 lines (7).

(maxbodylength)